src/Core/Domain/Category/SeoSettings.php

```diff
@@ -50,4 +50,9 @@ final class SeoSettings
      * Recommended length of SEO description
      */
     public const RECOMMENDED_DESCRIPTION_LENGTH = 160;
+
+    /**
+     * Maximum length of SEO keywords (value is constrained by database)
+     */
+    public const MAX_KEYWORDS_LENGTH = 255;
```

Comment:
same for these consts, make sure they have correct values based on object model and use these consts there as well

Reply:
This one seems correct based on 
https://github.com/PrestaShop/PrestaShop/blob/60756529c8aa7343c820db65b80767316948a49d/classes/Category.php#L134

But indeed this const should now replace the hard-coded value in `CategoryCore`

Reply:
Can you please replace in `classes/Category` the hard-coded values with the ones you added here? And same for the one in `CategorySettings` ideally